Jellyfin.Api/Helpers/TranscodingJobHelper.cs

```diff
@@ -323,6 +323,15 @@ private async Task KillTranscodingJob(TranscodingJobDto job, bool closeLiveStrea
         if (delete(job.Path!))
         {
             await DeletePartialStreamFiles(job.Path!, job.Type, 0, 1500).ConfigureAwait(false);
+            if (job.MediaSource?.VideoType == VideoType.Dvd || job.MediaSource?.VideoType == VideoType.BluRay)
+            {
+                var concatFilePath = Path.Join(_serverConfigurationManager.GetTranscodePath(), job.MediaSource.Id + ".concat");
+                if (File.Exists(concatFilePath))
+                {
+                    _logger.LogInformation("Deleting ffmpeg concat configuration at {Path}", concatFilePath);
+                    _fileSystem.DeleteFile(concatFilePath);
```

Comment:
```suggestion
                    File.Delete(concatFilePath);
```
_filesystem does way more work on windows more work in case the file is readonly, this shoeldn't be necessary here